en/rules.md

```diff
@@ -158,4 +158,4 @@ the following will **ALWAYS** be true for scorehunts.
  1. Scorehunt Information will be availible in #score-hunting & the [Public Score Hunt Spreadsheet](https://docs.google.com/spreadsheets/d/1iabjrE_O52rNifvUi8s1tBqzThGqa699pmwZdGZTCN8/edit?usp=sharing)
  2. Scorehunts will **NOT** allow CS changer.
  3. Scorehunts will follow the rules set here.
- 4. Breaking scorehunt rules will incur a normal moderation punishment (See Moderation Policy) + a blacklist from all future scorehunts.
+ 4. Breaking scorehunt rules will incur a normal moderation punishment (See Moderation Policy) + a 1 year blacklist from future scorehunts.
```

Comment:
can u make it not seem like the blacklist disappears after 1 year

Reply:
ex: "a blacklist appealable after 1 year?"

Reply:
should be fine enough

Reply:
> ex: "a blacklist appealable after 1 year?"

I mean we can change "blacklist" to "ban". It is written as blacklist because the original idea was that the the ban was permanent and not appelable but Im not the biggest fan of the idea anymore.